### PR TITLE
pretty-print Controls format  made Pretty

### PR DESCRIPTION
**Pull request by suhasgumma**
Pretty Print

## Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [x] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

### CODE REVIEW

**Comment by dwertent** [comment]
Nice. The URLs might be an [issue](https://github.com/kubescape/kubescape/issues/904).
 @YiscahLevySilas1 Do we have a convention for the CIS controls?

**Comment by kooomix** [comment]
@dwertent the doc controls naming conventions will be:
https://hub.armosec.io/docs/c-0001 - for none-cis control ids.
https://hub.armosec.io/docs/cis-1-1-3 - for cis control ids.

Will be fully active live within the next couple of days.

**Comment by suhasgumma** [comment]
Hi @dwertent, Changed the link conventions for CIS controls as suggested by @kooomix 

Pretty Print

**Review by dwertent — COMMENTED** [review]
How about we move this functionality to the utils directory so we can call it also when printing the results?
This will solve #904 .

**Comment by suhasgumma** [comment]
Hi @dwertent, 
There is already a `getControlLink` function in core/pkg/resultshandling/printer/v2 . Shall I update it?

File: core/pkg/resultshandling/printer/v2/prettyprinter.go [hunk 1]
```diff
@@ -255,8 +255,10 @@ func frameworksScoresToString(frameworks []reportsummary.IFrameworkSummary) stri
 	return ""
 }
 
-func getControlLink(controlID string) string {
-	return fmt.Sprintf("https://hub.armosec.io/docs/%s", strings.ToLower(controlID))
+func GetControlLink(controlID string) string {
```
Comment (dwertent) [inline core/pkg/resultshandling/printer/v2/prettyprinter.go:258]: This function should be in the [utils](https://github.com/kubescape/kubescape/tree/dev/core/cautils) package (you can create a new file for this) and not in the printer package.

**Review by dwertent — CHANGES_REQUESTED** [review]

File: core/pkg/resultshandling/printer/v2/prettyprinter.go (hunk 1 above)
Comment (suhasgumma) [reply core/pkg/resultshandling/printer/v2/prettyprinter.go]: > This function should be in the [utils](https://github.com/kubescape/kubescape/tree/dev/core/cautils) package (you can create a new file for this) and not in the printer package.

Hi @dwertent, 
Made the necessary changes!!

**Review by dwertent — APPROVED** [review]

**Comment by oakgul1** [comment]
Hi All i am using the latest image but the link still doenst work is shows 5.1.1 istead of the right one

**Comment by kooomix** [comment]
Hi @oakgul1 ,

What is the exact link you were trying?

**Pull request merged**